packages/ui-library/.eslintrc.js

```diff
@@ -11,4 +11,26 @@ module.exports = {
   plugins: ['@typescript-eslint'],
   extends: ['../eslint-config-boiler'],
   ignorePatterns: ['types/', '*.svg', '*.scss', '*.css', '*.md', '*.config.mjs', '.*', 'webpack*.js'],
+  rules: {
+    'no-restricted-imports': [
+      'error',
+      {
+        patterns: [
+          {
+            group: ['lit'],
+            importNamePattern: 'LitElement',
+            message: `Don't use the default LitElement class. Import from /utils/lit-element-custom instead`,
+          },
+        ],
+      },
+    ],
+  },
+  overrides: [
+    {
+      files: ['src/utils/lit-element-custom.ts'],
+      rules: {
+        'no-restricted-imports': 'off',
+      },
+    },
+  ],
```

Comment:
since for now this is the only exception it's probably simpler to just add a `// eslint-disable` comment in the referenced file instead. eslintrc overrides can easily get out of hand if not managed properly.

Reply:
Make sense, adjusted now.